docs/_articles/en/getting-started/java-setup.md

```diff
@@ -119,3 +119,13 @@ Windows:
   "salesforcedx-vscode-apex.java.home": "C:\\Program Files\\Java\\jdk1.8.0_131"
 }
 ```
+
+# Advanced Setup
+
+By default, the Apex Language Server process gets up to 1/4 of your system's physical memory allocated to it, that's how the Java JVM defaults works. There are certain situations, like working with large projects, in which you'll want to override that default and allocate more memory to the process. To do so you can use the `salesforcedx-vscode-apex.java.memory` setting and specify the maximum size, in megabytes, of the memory allocation. This value must be a multiple of 1024 greater than 2 MB.
```

Comment:
By default, a JVM allocates up to one fourth of  system's physical memory to the Apex Language Server process. If you are working on projects with more memory requirement, use salesforcedx-vscode-apex.java.memory setting to override the defaults. Use this setting to specify the maximum size of memory allocation in megabytes and in multiples of 1024 greater than 2 MB.

Reply:
I don't understand "multiples of 1024 greater than 2 MB." This value is defined in MB and should be multiples of 1024 (like 2048 or 4096 MB), why is there a requirement of "greater than 2 MB?"

Reply:
I agree with Jag, I think that statement would confuse customers too.

Reply:
Yeah now that you mention it we can remove it cause it doesn't make sense with this implementation. The background is, this setting is allowing you to set the maximum heap size for the java process (`-Xmx`) and that value supports bytes, megabytes, gigabytes, etc. That's where "multiples of 1024 greater than 2MB" comes from. But, since we implemented this to specifically set it in megabytes we can remove the "greater than 2MB" part.